app/shared/components/composition-search/LazyListItem.test.tsx

```diff
@@ -0,0 +1,42 @@
+import { render, screen } from '@testing-library/react';
+import React from 'react';
+
+import { VirtualizedListbox } from './LazyListItem';
+
+jest.mock('react-virtualized', () => {
+  return {
+    AutoSizer: ({ children }: any) => <div>{children({ width: 100 })}</div>,
+    List: ({ rowCount, rowRenderer }: any) => (
+      <div>{Array.from({ length: rowCount }).map((_, index) => rowRenderer({ index, key: index, style: {} }))}</div>
+    )
+  };
+});
+
+describe('VirtualizedListbox', () => {
+  test('renders children inside virtualized list', () => {
```

Comment:
```suggestion
  it('should render children inside virtualized list', () => {
```
We write tests in this code style, so please stick to it
And make changes in next one test too

Reply:
Thanks, will keep that in mind.